resources/inspectionDescriptions/ImpexMoreThanOnlyCodeAttributePresentForStaticEnumInspection.html

```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ This file is part of "SAP Commerce Developers Toolset" plugin for Intellij IDEA.
+  ~ Copyright (C) 2019 EPAM Systems <hybrisideaplugin@epam.com>
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation, either version 3 of the
+  ~ License, or (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  ~ See the GNU Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public License
+  ~ along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<html>
+<body>
+<p>More than only code attribute presents for static enum in ImpEx inspection</p>
```

Comment:
Sentence is not quite clear, "more than only code"
Not every dev may understand what is "static" enum and this sentence, it is rather non-dynamic or java enum.

Providing a code sample will be very nice, use html `code` + `pre` blocks for code samples.
Include link to docs if any